reference/metadata_tables.rst

```diff
@@ -43,16 +52,14 @@ The pg_dist_shard table stores metadata about individual shards of a table. This
 +----------------+----------------------+---------------------------------------------------------------------------+
 |      Name      |         Type         |       Description                                                         |
 +================+======================+===========================================================================+
-| logicalrelid   |           oid        | | Distributed table to which this shard belongs. This value references the|
+| logicalrelid   |         regclass     | | Distributed table to which this shard belongs. This value references the|
 |                |                      | | relfilenode column in the pg_class system catalog table.                |
 +----------------+----------------------+---------------------------------------------------------------------------+
 |    shardid     |         bigint       | | Globally unique identifier assigned to this shard.                      |
 +----------------+----------------------+---------------------------------------------------------------------------+
 | shardstorage   |            char      | | Type of storage used for this shard. Different storage types are        |
 |                |                      | | discussed in the table below.                                           |
 +----------------+----------------------+---------------------------------------------------------------------------+
-|  shardalias    |            text      | | Deprecated and unused column. Will be removed in a future release.      |
-+----------------+----------------------+---------------------------------------------------------------------------+
 | shardminvalue  |            text      | | For append distributed tables, minimum value of the distribution column |
```

Comment:
Not part of your review, but the [compound modifier](https://en.wikipedia.org/wiki/Compound_modifier) _append-partitioned_ should be hyphenated.